src/apphosting/app.ts

```diff
@@ -153,4 +153,4 @@ function isQuotaError(error: any): boolean {
   return code === 429;
 }
 
-export = webApps;
+export default webApps;
```

Comment:
default exports shouldn't (generally) be used: go/tsjs-style#exports

Reply:
why wouldn't it just be `export const webApps` way above?

Reply:
Changed to export const. 

I'm not sure why I can't respond to your other message, but the firebase-vscode targets module: "es2020" which restricts usage of export assignments.

Reply:
Hmm. firebase-tools's target could probably get updated one of these days too...